### PR TITLE
i18n(fr): Update all files `fr/basics/*` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update files : 
- [i18n(fr): Update astro-components.mdx](https://github.com/withastro/docs/commit/98e74a3507a3bbd6536e8cdf6cdef72a7c88a04e) 
- [i18n(fr): Update astro-pages.mdx](https://github.com/withastro/docs/commit/9dd92dd634fbf1188a55f5cfe7f043b4e5d9b156) 
- [i18n(fr): Update astro-syntax.mdx](https://github.com/withastro/docs/commit/0268389dae12267fd7d1f28a19b8c66666bfa0fe) 
- [i18n(fr): Update basics/layouts.mdx](https://github.com/withastro/docs/commit/ed06fe2619a220778cc8b47d9ed96a2188839dd1) 
- [i18n(fr): Update basics/rendering-modes.mdx](https://github.com/withastro/docs/commit/bcad266147b4a3e07c7a65a8a9fc809d468a057b)

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/basics/layouts.mdx [hunk 1]
```diff
@@ -14,7 +14,9 @@ Par convention, nous utilisons le terme "mise en page" pour les composants Astro
...
 
-Les composants de mise en pages sont généralement placés dans un dossier `src/layouts` dans votre projet pour l'organisation, mais ce n'est pas une obligation; vous pouvez choisir de les placer n'importe où dans votre projet. Vous pouvez même placer des composants de mise en page à côté de vos pages en [préfixant les noms de mise en page par `_`](/fr/guides/routing/#exclure-des-pages).
+Cependant, si un composant de mise en page contient une coquille de page, son élément `<html>` doit être le parent de tous les autres éléments du composant. Tous les éléments [`<style>`](/fr/guides/styling/#styliser-avec-astro) ou [`<script>`](/fr/guides/client-side-scripts/#utilisation-du-script-dans-astro) doivent être entourés par les balises `<html>`.
```
Comment (pioupia) [inline src/content/docs/fr/basics/layouts.mdx:17]: Une coquille ? Je t'avoue que je peine à trouver des traductions adéquates, mais une "enveloppe" me parait plus appropriée.
```suggestion
Cependant, si un composant de mise en page contient une enveloppe de page, son élément `<html>` doit être le parent de tous les autres éléments du composant. Tous les éléments [`<style>`](/fr/guides/styling/#styliser-avec-astro) ou [`<script>`](/fr/guides/client-side-scripts/#utilisation-du-script-dans-astro) doivent être entourés par les balises `<html>`.
```

**Review by pioupia — CHANGES_REQUESTED** [review]

File: src/content/docs/fr/basics/layouts.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/basics/layouts.mdx:17]: C'est mieux! Merci @pioupia !

**Review by pioupia — APPROVED** [review]
LGTM!

**Review by dreyfus92 — APPROVED** [review]
Merci @thomasbnt, LGTM ✅

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work everyone! 💜

**Pull request merged**